src/components/Body.jsx

```diff
@@ -0,0 +1,40 @@
+import Header from "./Header";
+import NavBar from "./NavBar";
+import Users from "./Users";
+import AllArticles from "./Articles";
+
+function Body() {
```

Comment:
? what is this component? We want to try and keep merged code relevant and ready for production